src/contracts/libraries/GPv2Interaction.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.7.6;
+
+/// @title Gnosis Protocol v2 Interaction Library
+/// @author Gnosis Developers
+library GPv2Interaction {
+    /// @dev Interaction data for performing arbitrary contract interactions.
+    /// Submitted to [`GPv2Settlement.settle`] for code execution.
+    struct Data {
+        address target;
+        uint256 value;
+        bytes callData;
+    }
+
+    /// @dev Execute an arbitrary contract interaction.
+    ///
+    /// @param interaction Interaction data.
+    function execute(Data calldata interaction) internal {
```

Comment:
Wait... was the singular execution moved here? Seems natural.

Reply:
Yes! YOu now execute an interaction with `interaction.execute()` instead of `executeInteraction(interaction)`.